src/scss/_main.scss

```diff
@@ -464,6 +464,11 @@
 			width: oGridGetMaxWidthForLayout($o-grid-fixed-layout, $grid-mode: 'fixed');
 		}
 	}
+
+	&--bleed {
```

Comment:
Because of the way the mixin is being used here and the class it would need to generate I think this is ok